pkg/apis/maistra/v2/runtime.go

```diff
@@ -199,13 +199,19 @@ type CommonPodRuntimeConfig struct {
 // XXX: istio does not support full corev1.Affinity settings, hence the special
 // types here.
 type Affinity struct {
+	// +optional
+	NodeAffinity *corev1.NodeAffinity `json:"nodeAffinity,omitempty"`
```

Comment:
If we add this, will we need to modify the rest of the charts to add support for this (and PodAffinity below)?

Reply:
No, I've added validation to the webhook to prevent users from setting these fields for any other component than Kiali.